controllers/users.controller.mjs

```diff
@@ -1,37 +1,34 @@
-import jsSHA from 'jssha';
+import jsSHA from "jssha";
 
 const SALT = process.env.SALT_PASSWORD;
 
 const getHashSalted = (input) => {
   // create new SHA object
-  const shaObj = new jsSHA('SHA-512', 'TEXT', { encoding: 'UTF8' });
+  const shaObj = new jsSHA("SHA-512", "TEXT", { encoding: "UTF8" });
   // create an unhashed cookie string based on user ID and salt
-  const unhashedString = `${input}-${SALT}`;
+  const unhashedString = SALT == undefined ? `${input}` : `${input}-${SALT}`;
```

Comment:
I added this line to check whether we have SALT_PASSWORD in our .env file

Reply:
Okay!